src/Infrastructure/Persistence/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -238,6 +238,307 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.ToTable("trust_levels");
                 });
+
+            modelBuilder.Entity("Codidact.Infrastructure.Identity.ApplicationUser", b =>
+                {
+                    b.Property<string>("Id")
+                        .HasColumnName("id")
+                        .HasColumnType("text");
+
+                    b.Property<int>("AccessFailedCount")
+                        .HasColumnName("access_failed_count")
+                        .HasColumnType("integer");
+
+                    b.Property<string>("ConcurrencyStamp")
+                        .IsConcurrencyToken()
+                        .HasColumnName("concurrency_stamp")
+                        .HasColumnType("text");
+
+                    b.Property<string>("Email")
+                        .HasColumnName("email")
+                        .HasColumnType("character varying(256)")
```

Comment:
Emails have a max length of 320 characters - 256 isn't enough for an email field.